setup.py

```diff
@@ -109,7 +109,7 @@
 setup_requires = ['numpy>=1.9']
 
 # pretty much needed
-install_requires = ['numpy>=1.9', 'qtpy>=1.1', 'setuptools>=1.0',
```

Comment:
Supposedly `qtpy` does not require Qt to be installed.  But it's a good package to avoid having to support many different versions of qt 4/5/pyside (and probably qt 6 when it comes out).